guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -5,8 +5,9 @@ You can use the Leapp tool to upgrade as well as to help detect and resolve issu
 
 .Prerequisites
 ifdef::satellite[]
-* Review Known Issues before you begin an upgrade.
-For more information, see {ReleaseNotesDocURL}ref_known-issues_assembly_introducing-red-hat-satellite[Known Issues in {ProjectName} {ProjectVersion}].
+* Review known issues before you begin an upgrade.
+** {ReleaseNotesDocURL}known-issues[Known issues in {ProjectName} {ProjectVersion}].
+** {RHELDocsBaseURL}9/html-single/upgrading_from_rhel_8_to_rhel_9/index#known-issues_troubleshooting[Known issues for the {EL} 8.10 to RHEL 9.4 upgrade]. 
```

Comment:
Do we want to be specific in the RHEL versions? If we keep them open ended we won't have to update them when RHEL releases. @evgeni thoughts?

Reply:
I'd keep it at "8 to 9", not mentioning the minor versions

Reply:
It's actually the target document -- RHEL upgrade guide -- that is specific about the RHEL versions: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/upgrading_from_rhel_8_to_rhel_9  So if we really want to link to that, we can't really avoid mentioning the minor versions.

However, I'm wondering whether linking to a known issues section is the right move. The [Jira ticket](https://issues.redhat.com/browse/SAT-17113) seems to request linking to a section for preparing RHEL for the Leapp upgrade instead, which would be this link: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/upgrading_from_rhel_8_to_rhel_9/index#preparing-a-rhel-8-system-for-the-upgrade_upgrading-from-rhel-8-to-rhel-9.

@AkshayGadhaveRH Can you please take one more look at the Jira ticket description and double-check what link needs to be added?

Reply:
Wait, right, the request is to refer to the RHEL upgrade guide for HTTP proxy setup. Which is indeed in the KI sections for specific minor releases: [9.3. Known issues for the RHEL 8.10 to RHEL 9.4 upgrade](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/upgrading_from_rhel_8_to_rhel_9/index#known-issues_troubleshooting) and [9.4. Known issues for the RHEL 8.8 to RHEL 9.2 upgrade](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/upgrading_from_rhel_8_to_rhel_9/index#known-issues-from-8.8_troubleshooting). I don't really see a way to avoid mentioning the minor releases if they are included in the section heading.

Reply:
To continue my monologue: I asked the RHEL upgrade guide maintainer for details and she recommends linking to the assembly: https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/upgrading_from_rhel_8_to_rhel_9/index#troubleshooting-resources_troubleshooting
The individual modules can change over time based on differences in the information for specific upgrade paths so she doesn't recommend linking to them.

However @AkshayGadhaveRH if you decide to link to the assembly, you might need to introduce the link with a few words on what you want users to look for (HTTP Proxy scenario, based on the Jira ticket).

Reply:
@asteflova added a step for this.